CMakeLists.txt

```diff
@@ -295,6 +296,18 @@ if(BUILD_C)
         OUTPUT_NAME retro
         CXX_VISIBILITY_PRESET default)
     target_link_libraries(retro-c retro-base ${STATIC_LDFLAGS})
+    install(TARGETS retro-c)
+    file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/retro-c.pc
+"
```

Comment:
I'd strongly recommend using `configure_file` and a .in file instead of doing this inline.